apis/v1alpha2/shared_types.go

```diff
@@ -84,6 +86,35 @@ type ParentRef struct {
 	//
 	// +optional
 	SectionName *SectionName `json:"sectionName,omitempty"`
+
+	// Port is the network port this Route targets. It can be interpreted
+	// differently based on the type of parent resource:
+	//
+	// * Gateway: All listeners listening on the specified port that also
+	// support this kind of Route(and select this Route). It's not
```

Comment:
This sentence is a little unclear.  Is it a definition of the Gateway interpretation of Port?  Or is Gateway a type of parent resource?

Reply:
Good catch - will be fixed by #1160

Reply:
Updated, thanks!